File: restaurant-app/src/app/model/chart-generate-data.ts
import {ChartType} from "./chart-type";
import {OrderType} from "./order-type";
import {PeriodType} from "./period-type";

export interface ChartGenerateData {
  periodType: PeriodType;
  period: any;
  placeId: any;
  chartType: ChartType;
  orderType: OrderType;
}

File: restaurantservice/order-service/src/main/java/pl/restaurant/orderservice/api/response/ChartGenerateData.java
package pl.restaurant.orderservice.api.response;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ChartGenerateData {
    private PeriodType periodType;
    private String period;
    private Long placeId;
    private ChartType chartType; //think about it
    private OrderType orderType;
}
